src/llfs/volume_config.test.cpp

```diff
@@ -180,24 +180,24 @@ TEST_F(VolumeConfigTest, ConfigRestore)
 
     llfs::Volume& volume = **maybe_volume;
     llfs::StatusOr<batt::Grant> grant =
-        volume.reserve(volume.calculate_grant_size("alpha") +        //
-                           volume.calculate_grant_size("bravo") +    //
-                           volume.calculate_grant_size("charlie") +  //
-                           volume.calculate_grant_size("delta"),
+        volume.reserve(volume.calculate_grant_size(std::string_view("alpha")) +        //
```

Comment:
We already have `calculate_grant_size` overload that takes `const std::string_view&`. The compatibility broke because we added a templated calculate_grant_size.

Reply:
Right... I think that's due to the reason I mentioned in my analogous comment on `Volume::append`.  The compiler is probably considering the template to be a better match than the `std::string_view` overload because it can infer `T` as `const char *` and by so doing, avoid having to perform an implicit conversion by constructing a `std::string_view` temporary for the string.  In both cases, I think the solution is to use `std::enable_if_t` to specifically exclude this edge case.

Reply:
As discussed will keep this part same and get rid of `calculate_grant_size(const std::string_view&..)`.